internal/awsnuke/client_integration_test.go

```diff
@@ -0,0 +1,89 @@
+package awsnuke_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/aws-nuke/internal/awsnuke"
+)
+
+var AWSAccountId = "353720269506"
```

Comment:
Might want to call it `DefaultAwsAccountId` and add a comment that this is Gruntwork's "Nuclear Wasteland" account for testing.